File: C++/Linked list/Finding out Duplicate elements/Code.py
#include<stdio.h>

struct Node
{
    int data;
    struct Node*next;
}*first=NULL;

void Create(int A[],int size)
  {
      struct Node*t,*last;
      first=(struct Node*)malloc(sizeof(struct Node));
      first->data=A[0];
      first->next=NULL;
      last=first;
  int i;
  for(i=1;i<size;i++)
      {
        struct Node*t=(struct Node*)malloc(sizeof(struct Node));
        t->data=A[i];
        t->next=NULL;
        last->next=t;
        last=t;
      }
  }

void Display(struct Node*ptr)
  {
      while(ptr!=NULL)
         {
           printf("%d  ",ptr->data);
           ptr=ptr->next;
         }
  }

File: C++/Linked list/7) Inserting node/Creating a LL by inserting at the last/Code.c
#include<stdio.h>
#include<stdlib.h>

struct Node
{
	int data;
	struct Node *next;	
}*first=NULL,*last=NULL;

Insert(int element)
{
	struct Node *t=(struct Node*)malloc(sizeof(struct Node));
	t->data=element;
	t->next=NULL;
	
	// If No Nodes are created
	if(first==NULL) 	{ first=last=t;	}
	
	// If Nodes are created
	else 
	{
		last->next=t;
		last=t;		
	}
}

void Display(struct Node *ptr)
{
	while(ptr!=NULL)
	{
		printf("%d ",ptr->data);
		ptr=ptr->next;
	}
}

int main()
{
	Insert(4);
	Insert(5);
	
	Display(first);
}
